### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by adem4ik**

### CODE REVIEW

File: avidemux_ru.ts [hunk 1]
```diff
@@ -1307,42 +1307,42 @@ Drop timing informations ?</source>
...
         <location line="+1"/>
         <source>60</source>
-        <translation type="unfinished"></translation>
+        <translation>60</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>_Assign following standard frame rate:</source>
-        <translation type="unfinished"></translation>
+        <translation>_Назначить следующее стандартной частотой кадров:</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: The English source can be rephrased as "use the following standard frame rate when making a video out of a set of sequentially named images", if this helps to find an adequate translation. Any of the offered fps values are already standard fps values in the world, not related to our settings. We just choose one.

File: avidemux_ru.ts [hunk 2]
```diff
@@ -9565,7 +9565,7 @@ I will assume it has and try to continue</source>
     <message>
         <location filename="../../../avidemux_plugins/ADM_demuxers/Matroska/ADM_mkvIndexer.cpp" line="+630"/>
         <source>Indexing Matroska Video Track</source>
-        <translation type="unfinished"></translation>
+        <translation>Индексация видеодорожки Matroska</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: Would it be a total no-go to use the original Russian word for Matroska? I can expand the compressed English source to its full meaning as "Indexing the video track in a Matroska container" (acutely missing either a preposition or congruence in the translation).

File: avidemux_ru.ts [hunk 3]
```diff
@@ -10690,27 +10690,27 @@ Only VP8/VP9/AV1 video and Vorbis/Opus audio supported</source>
...
         <location line="+14"/>
         <source>Color primaries:</source>
-        <translation type="unfinished"></translation>
+        <translation>Первичные цвета:</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Transfer characteristics:</source>
-        <translation type="unfinished"></translation>
+        <translation>Характеристики перехода:</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: Are you sure this is a valid equivalent for "transfer characteristics"? I would suggest adhering to https://ru.wikibooks.org/wiki/%D0%A6%D0%B8%D1%84%D1%80%D0%BE%D0%B2%D0%BE%D0%B5_%D0%BF%D1%80%D0%B5%D0%B4%D1%81%D1%82%D0%B0%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5_%D1%86%D0%B2%D0%B5%D1%82%D0%B0 => "функция преобразования"

File: avidemux_ru.ts [hunk 4]
```diff
@@ -11476,188 +11476,188 @@ The current editing has been saved and will be reloaded at next start</source>
...
         <location line="+31"/>
         <source>Unknown</source>
-        <translation type="unfinished"></translation>
+        <translation>Неизвестно</translation>
     </message>
     <message>
         <location line="-98"/>
         <source>Limited (MPEG)</source>
-        <translation type="unfinished"></translation>
+        <translation>Ограниченное (MPEG)</translation>
```
Comment (eumagga0x2a) [inline avidemux_ru.ts]: This refers to color range, which is translated above as "Цветовой диапазон". Could you please fix the congruence? Maybe also for the adjacent translations as well.

**Review by eumagga0x2a — COMMENTED** [review]
Thank you for the update, I'll merge it once the questions are resolved.

File: avidemux_ru.ts (hunk 1 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Fixed

File: avidemux_ru.ts (hunk 2 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Original Russian word isn't used at all across Russian speakers unfortunatly :/ Take a look at wiki article https://ru.wikipedia.org/wiki/Matroska — it is "Matroska" everywhere. Anyway I've rephrased the translation.

File: avidemux_ru.ts (hunk 3 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Fixed, thanks!

File: avidemux_ru.ts (hunk 4 above)
Comment (adem4ik) [reply avidemux_ru.ts]: Fixed now.

**Comment by eumagga0x2a** [comment]
Thanks!

**Pull request merged**